README.md

```diff
@@ -31,10 +31,14 @@ make services-invoke-publish
 ## Dev setup
 
 ```sh
+# Make sure Ruby is installed
```

Comment:
You still require python in that instance though so it's the same as using Ruby?